File: 23.10-Spring-Security/springBoot/src/main/java/com/security/springboot/jwt/AuthConstants.java
package com.security.springboot.jwt;

public interface AuthConstants {
    String AUTH_HEADER = "Authorization";
    String TOKEN_TYPE = "Bearer "; // 띄어 쓰기가 있어야 한다.
    String COOKIE_HEADER = "Set-Cookie";
    String REFRESH_TOKEN_PREFIX = "refresh_token";
}

File: kotlin/src/main/kotlin/com/example/kotlin/security/AuthConstants.kt
package com.example.kotlin.security

object AuthConstants {
    const val AUTH_HEADER = "Authorization"
    const val TOKEN_TYPE = "Bearer " // 띄어 쓰기가 있어야 한다.
    const val COOKIE_HEADER = "Set-Cookie"
    const val REFRESH_TOKEN_PREFIX = "refresh_token"
}